apps/services/auth/api/src/app/v1/delegations/delegations.controller.spec.ts

```diff
@@ -284,6 +318,24 @@ describe('DelegationsController', () => {
                 ),
               ).toBeTruthy()
             })
+
+            it('should have deleted prModels and prRightModels for deceased persons', async () => {
```

Comment:
Let's discuss if we want to delete personal representative data for individuals that we assume are "deceased". Might want to just "hide" these delegations without deleting.

On one hand, we do not "own" these rows. Réttindagæslan should do this. But we don't know if they will, and if these will collect up and cause weird behaviour later on.

/cc @hrefnalind

Reply:
Yes I think it makes more sens to hide them